proof/invariant-abstract/AARCH64/ArchBCorres_AI.thy

```diff
@@ -45,7 +45,7 @@ crunch prepare_thread_delete
 
 end
 
-requalify_facts AARCH64.arch_finalise_cap_bcorres AARCH64.prepare_thread_delete_bcorres
+arch_requalify_facts arch_finalise_cap_bcorres prepare_thread_delete_bcorres
 
 declare arch_finalise_cap_bcorres[wp] prepare_thread_delete_bcorres[wp]
```

Comment:
It's not clear to me what you're using to decide when requalifications in Arch files can be moved to generic ones. For instance, why not here?

Reply:
This can definitely be moved to the theory that includes this one, I probably only missed it because it's BCorres and was super early in the run-through. If you see others, let me know. Will add FIXME.

Reply:
I'll add comments for the others I saw, although I haven't stopped to think about whether they actually make sense in the Arch file or can be moved.

Reply:
moved